nilearn/datasets/atlas.py

```diff
@@ -616,3 +616,40 @@ def fetch_atlas_aal(version='SPM12', data_dir=None, url=None, resume=True,
               'labels': labels_dict}
 
     return Bunch(**params)
+
+
+def fetch_atlas_dosenbach_2010():
+    """Download and load the Dosenbach et al. brain atlas composed of 160 ROIs
+    grouped in 6 networks.
```

Comment:
> The difference form many other fetchers appears to be gthat the atlas is a set
> of coordinates and not 3D shapes. That might be mentioned here.

I do wonder if we should call this an atlas? Maybe fetch_coords_dosenbach
rather?